src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,40 +1,64 @@
 import React from 'react';
+import { useAppSelector } from '../../app/hooks';
+import { useDispatch } from 'react-redux';
+import { filterSlice } from '../../features/filter';
+import { Status } from '../../types/Status';
 
 export const TodoFilter: React.FC = () => {
+  const filter = useAppSelector(state => state.filter);
+  const query = filter.query;
+  const dispatch = useDispatch();
+
+  const handleSelect = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    dispatch(filterSlice.actions.setStatus(event.target.value));
+  };
+
+  const ChangeQuery = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
```suggestion
  const changeQuery = (event: React.ChangeEvent<HTMLInputElement>) => {
```
I guess it was accidentally, but you should use camelCase